docs/docker/run.md

```diff
@@ -0,0 +1,41 @@
+## Run you own __Cardano Node__ 
+
+## OS Requirements
+
+- "docker-ce" installed.
+
+## Private mode
+
+### use cases
+
+- Pool Management
+- Wallet Management
+- Node testing
+
+```bash
+docker run -dit 
+-e NETWORK=mainnet 
+--name <YourCName>
+-p <your_custom_path>:/opt/cardano/cnode/priv
+-p <your_custom_db_path>:/opt/cardano/cnode/db
```

Comment:
-p for exposing ports -v for attaching/binding volumes.

Reply:
thanks for pointing out